.changeset/odd-parents-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-api-docs': minor
```

Comment:
Not a breaking change right? patch in that case

Reply:
No it isn't. I changed all changesets to `patch`.